### ISSUE TITLE
List of tweaks and fixes on Next js version

### ISSUE THREAD

**Issue by daviddubrino** [tomswain84/daskeyboardsite#5]
Here are the items I see at the current state and will continue to add items as it progresses:

- [ ] When you click "Learn More" from the Free Shipping modal in the header, the box should go away:
Vercel.app: 
![image](https://user-images.githubusercontent.com/14006273/172471106-946da25a-44f6-4e90-ac8e-9e5c421dd2fd.png)
4Eighty:
![image](https://user-images.githubusercontent.com/14006273/172471144-58244f32-53c5-4290-97e1-8b2d2e2e360f.png)

- [ ] Contact Us link should open in the same tab:
![image](https://user-images.githubusercontent.com/14006273/172471321-0e25f2f7-2f82-47f2-b06b-cb26f1c767a2.png)

- [ ] Spacing off on down arrows and text in the top nav and overlap with text in Chrome on macOS
Vercel.app: 
![image](https://user-images.githubusercontent.com/14006273/172470591-c07ff00f-64e2-422d-9f82-7fddda4170b8.png)
4Eighty: 
![image](https://user-images.githubusercontent.com/14006273/172470861-a69bff69-09b0-43b3-9155-01ac9b3f1be3.png)

- [ ] Replacing the D here as discussed
![image](https://user-images.githubusercontent.com/14006273/172471925-ec447536-b220-4591-836a-fe23c737b646.png)

**Comment by daviddubrino** [comment]
On mobile: 

- [ ] The reviews section is not formatted correctly:
vercel.app:
![image](https://user-images.githubusercontent.com/14006273/172473182-96953105-8aa3-4c3a-8721-e8966093c2e9.png)

4eighty: 
![image](https://user-images.githubusercontent.com/14006273/172473230-087c63b0-2e09-47f5-96b3-07dc40af78c8.png)

**Comment by mrth2** [comment]
@daviddubrino  please check the NextJS site again, now all issues above are resolved 
> https://dascommerce.vercel.app/

**Comment by daviddubrino** [comment]
Looks great @mrth2, except I'm not seeing the new D graphic. Should I close this ticket and create another just for that?

**Comment by daviddubrino** [comment]
Yes @josephcotten, we can put the original D back in for now and enter it as a new design request. Thanks!

**Comment by mrth2** [comment]
@daviddubrino original D image is now back

**Comment by daviddubrino** [comment]
Great, thanks
